aptos-move/aptos-vm/src/block_executor/mod.rs

```diff
@@ -143,17 +145,8 @@ impl ParallelAptosVM {
                     .collect()
             });
 
-        let log_context = AdapterLogSchema::new(state_view.id(), 0);
```

Comment:
why this is removed?

Reply:
moved at the call-site in aptos_vm.rs, where we already have the logger, and now will log no matter how we execute:
https://github.com/aptos-labs/aptos-core/blob/82968ffdf6e772b7fa26eae01770662473773193/aptos-move/aptos-vm/src/aptos_vm.rs#L974